### PR TITLE
Incorrect names of functions. open logic incorrect on windows

### PR DESCRIPTION
**Pull request by chrisnojima**
refactor to the kbfs actions caused this to not function on windows. unclear how it worked on osx (but it did). 

- [x] fixes exported names being incorrect
- [x] fixed borked windows open logic

@keybase/react-hackers

### CODE REVIEW

**Comment by zanderz** [comment]
Oh, this is why clicking a favorite wasn't working again?
LGTM :thumbsup:

**Comment by cjb** [comment]
@chrisnojima could we find out more about when it broke / how it worked?  need to know if there are broken builds out there

**Comment by zanderz** [comment]
Should I wait for this to be merged before making another release?

**Comment by chrisnojima** [comment]
i think this'll be fixed soon so we could defer a release. widget is pretty broken w/o this

**Comment by chrisnojima** [comment]
@cjb this seems to work fine on osx even with this (not exactly sure how). this got broken by this pr: https://github.com/keybase/client/pull/3179/files#diff-f7dabd05c1cd1f40d7ffcc77c3e89a15L22

**Comment by cjb** [comment]
@chrisnojima :thumbsup: thanks.  let's push a release with this fix and my menubar fix today, that'll avoid yesterday's build getting promoted.

File: shared/actions/kbfs/platform.desktop.js [hunk 1]
```diff
@@ -32,30 +32,33 @@ export function platformOpenInKBFS (path: string = Constants.defaultKBFSPath): A
 
     // On windows the path isn't /keybase
     // We can figure it out by looking at the extendedConfig though
-    if (process.platform === 'win32' && kbfsPath === Constants.defaultKBFSPath) {
-      const extendedConfig = Promise.resolve(state.config.extendedConfig)
+    if (process.platform === 'win32') {
+      if (kbfsPath === Constants.defaultKBFSPath) {
```
Comment (chrisnojima) [inline shared/actions/kbfs/platform.desktop.js]: if we haven't resolved the path, try to

File: shared/actions/kbfs/platform.desktop.js [hunk 2]
```diff
@@ -32,30 +32,33 @@ export function platformOpenInKBFS (path: string = Constants.defaultKBFSPath): A
...
-      }).then(kbfsPath => {
-        dispatch({type: Constants.changeKBFSPath, payload: {path: kbfsPath}})
+          return Promise.resolve(kbfsPath + '\\')
+        }).then(kbfsPath => {
+          dispatch({type: Constants.changeKBFSPath, payload: {path: kbfsPath}})
+          open(kbfsPath, path)
+        }).catch(e => {
+          console.warn('Error in parsing kbfsPath: ', e)
+        })
+      } else {
```
Comment (chrisnojima) [inline shared/actions/kbfs/platform.desktop.js]: this was missing. if we have resolved the path (like k:) then just open it, but don't use the default below

**Comment by chrisnojima** [comment]
@cjb take one more peek please

File: shared/actions/kbfs/platform.native.js [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 import type {AsyncAction} from '../../constants/types/flux'
 
-export function platformOpenInKBFS (path: string = ''): AsyncAction {
+export function openInKBFS (path: string = ''): AsyncAction {
```
Comment (cjb) [inline shared/actions/kbfs/platform.native.js]: Why this change?

File: shared/actions/kbfs/platform.desktop.js (hunk 2 above)
Comment (cjb) [inline shared/actions/kbfs/platform.desktop.js]: Are you saying that open in KBFS was broken on Windows for more than one reason, then?  How did it work before?

File: shared/actions/kbfs/platform.native.js (hunk 3 above)
Comment (chrisnojima) [inline shared/actions/kbfs/platform.native.js]: this is the original bug: https://github.com/keybase/client/blob/master/shared/actions/kbfs/index.js#L7 it expects an export called 'openInKBFS' which it renames to platformOpenInKBFS (since its delegating this out)

File: shared/actions/kbfs/platform.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline shared/actions/kbfs/platform.desktop.js]: i don't think its worked for awhile reliably. you can get this to work once (and it'll try and open it twice)

**Comment by cjb** [comment]
Okay, :ship: :thumbsup:
